### PR TITLE
Replaced JWST references with Roman

### PR DESCRIPTION
**Pull request by dmggh**
Replaced references to JWST with references to Roman in the readthedocs Roman Cal documentation.  Some areas will need embellishment or clarification as code development progresses.

### CODE REVIEW

File: docs/roman/introduction.rst [hunk 1]
```diff
@@ -262,11 +245,11 @@ individual steps, you have two options:
     This option will save the step results using the name specified.
 
 For example, to save the result from the dark current step of
-``calwebb_detector1`` in a file named based on ``intermediate``, use
+``calroman_detector1`` in a file named based on ``intermediate``, use
```
Comment (ddavis-stsci) [inline docs/roman/introduction.rst:248]: Does this add the ".asdf" or is the file just called "intermediate"? If the asdf is not added can we just call the file intermediate.asdf to be clear?

**Review by ddavis-stsci — APPROVED** [review]

**Comment by dmggh** [comment]
I redid the readthedocs build, which built successfully. But in the built docs it appears that some of the JWST->Roman changes I made to introduction.rst and other files do not appear in the built version. I do not understand why, so I will look into this.

**Comment by dmggh** [comment]
I wiped the readthedocs environment and did a rebuild, which appears to solve the problem. Now the built document contains my latest updates - all of the previous JWST references have been correctly replaced. Unless someone has any objections, I think this completes what is needed for RCAL-18. The built docs are at : https://roman-cal-pipeline.readthedocs.io/en/latest/index.html

**Comment by ddavis-stsci** [comment]
Can you also remove the CGI references in flat_selection.inc?
Thanks,
    Dave

**Comment by dmggh** [comment]
Yes, I think that is already been noticed for RCAL-49.


________________________________

**Pull request merged**